docs/confluence/perplexed.md

```diff
@@ -77,15 +79,29 @@ At this point, if sanity passed and in reasonable times (not more than 1-2 minut
     
 5.  Check one rule at a time and try to increase `-depth=15`. This could help if the code contains many branches.
     
-6.  It could be that internal function summaries were not applied, so if you provided those and there is no change in run times, it could be there’s a muted error there. ([thomas](https://certora.atlassian.net/wiki/people/60919dc2c87b550069bccbd4?ref=confluence) can you say how to debug this?)
+6.  It could be that internal function summaries were not applied, so if you provided those and there is no change in run times, please report to Certora.
```

Comment:
There's a new option that lists the summarization candidates (`-showInternalMethods`); we might suggest that people check that list.  There's a PR for documenting the flag: https://github.com/Certora/Documentation/pull/34